pages/cart.html

```diff
@@ -0,0 +1,125 @@
+          <span>Wishlist</span>
+        </div>
+      </div>
+    </nav>
+
+    <main class="cart">
+      <!-- <div class="container"> -->
+      <!-- <h3 class="h3 ta-center mb-6">Cart (1)</h3> -->
+
+      <div class="cart-container">
+        <div class="items">
+          <div class="cart-item">
+            <div class="cover-img">
+              <img
+                class="cart-item-img"
+                src="../assets/products-img/Nautica-Men-Tshirts-1.webp"
+                alt="first tshirt"
+              />
+            </div>
+            <div class="card-content">
+              <h4 class="card-title">Men Tshirt</h4>
+              <div class="product-price">
+                <span>Rs. 799</span>
+                <span>Rs. 999</span>
+                <span>(40% OFF)</span>
+              </div>
+              <p>
+                Quantity: <i class="px-1 fa-solid fa-circle-minus"></i>
+                1
+                <i class="fa-solid fa-circle-plus"></i>
+              </p>
+              <button class="btn">Remove from Cart</button>
+              <button class="btn">Move to wishlist</button>
+            </div>
+          </div>
+          <div class="cart-item">
+            <div class="cover-img">
+              <img
+                class="cart-item-img"
+                src="../assets/products-img/-Women-Kurta-Sets2-1.webp"
+                alt="first tshirt"
+              />
+            </div>
+            <div class="card-content">
+              <h4 class="card-title">Women Kurti</h4>
+              <div class="product-price">
+                <span>Rs. 1799</span>
+                <span>Rs. 2999</span>
+                <span>(40% OFF)</span>
+              </div>
+              <p>
+                Quantity: <i class="fa-solid fa-circle-minus"></i>
+                1
+                <i class="fa-solid fa-circle-plus"></i>
+              </p>
+              <button class="btn">Remove from Cart</button>
+              <button class="btn">Move to wishlist</button>
+            </div>
+          </div>
+        </div>
+
+        <div class="price-details-container">
+          <p class="price-details-title bold">PRICE DETAILS</p>
+          <hr />
+          <ul class="price-list">
+            <li class="price-item">
+              Price (2 items) <span>&#8377; 1999</span>
+            </li>
+            <li class="price-item">Discount <span>- &#8377; 1000</span></li>
+            <li class="price-item">
+              Delivery Charges <span>&#8377; 300</span>
+            </li>
+            <hr />
+            <li class="price-item bold">
+              TOTAL AMOUNT <span>&#8377; 2299</span>
+            </li>
+            <hr />
+          </ul>
+          <p class="discount-txt">You will save &#8377; 1000 on this order</p>
+          <button class="btn primary-btn">PLACE ORDER</button>
+        </div>
+      </div>
+    </main>
+
+    <script
```

Comment:
You can place the script tag of fontawesome in the head. As the icons will not load until the whole page is parsed.

Reply:
thank you for this info